schemas/telemetry/voice/voice.4.schema.json

```diff
@@ -1218,23 +1236,29 @@
           "title": "Was the intent action successful?",
           "type": "boolean"
         },
+        "intentTime": {
+          "description": "Milliseconds between getting the utterance and execution the intent",
+          "title": "Time to execute the intent",
+          "type": "number"
+        },
         "internalError": {
           "title": "If there was an unexpected error, the name of that error",
           "type": "string"
         },
-        "serverErrorIntentParser": {
```

Comment:
Unfortunately, schemas are append/update only. Removing columns from a table is not well defined, and requires dropping all of the existing data or updating the version number of the schema.

Reply:
Fixed. FYI, serverTimeIntentParser moved during the rebuilds, so it appears deleted but is inserted elsewhere.